modules/apps/29-fee/keeper/escrow.go

```diff
@@ -38,6 +39,17 @@ func (k Keeper) EscrowPacketFee(ctx sdk.Context, identifiedFee types.IdentifiedP
 
 	// Store fee in state for reference later
 	k.SetFeeInEscrow(ctx, identifiedFee)
+
+	// Emit event so that relayers know an incentivized packet is ready to be relayed
+	ctx.EventManager().EmitEvent(
+		sdk.NewEvent(
+			types.EventTypeSendIncentivizedPacket,
+			sdk.NewAttribute(channeltypes.AttributeKeyPortID, identifiedFee.PacketId.PortId),
+			sdk.NewAttribute(channeltypes.AttributeKeyChannelID, identifiedFee.PacketId.ChannelId),
+			sdk.NewAttribute(channeltypes.AttributeKeySequence, fmt.Sprint(identifiedFee.PacketId.Sequence)),
```

Comment:
We need to emit the fee for each relayer msg:

`RecvFee => recv_coins`
`AckFee => ack_coins`
`TimeoutFee => timeout_coins`